src/azure-cli-core/setup.py

```diff
@@ -49,6 +49,8 @@
     'colorama~=0.4.1',
     'humanfriendly>=4.7,<10.0',
     'jmespath',
+    'configobj',
```

Comment:
Python has its own [`configparser`](https://docs.python.org/3/library/configparser.html). We usually avoid using external dependencies whenever possible.

Reply:
I import `configobj` here because `configparser` can't support sub section.
```
[section]
    [[sub section]]
```
But if we use JSON instead of INI, then this dependency can be removed.